bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js

```diff
@@ -1713,7 +1731,151 @@ var WebApp = (function() {
 			}
 		}
 	}
-
+	
+	function Slider(item) {
+		var _t = this;
```

Comment:
I just wanted something to refer to instance of Slider in nested scopes. Sometimes `that` is used as a variable name. Should I change it to `that` or something other?